modules/Project/Save.php

```diff
@@ -133,4 +133,4 @@
 	$_REQUEST['return_url'] = "index.php?module=Project&action=view_GanttChart&project_id=" . $return_id;
     handleRedirect($return_id,'Project');
 }
-?>
\ No newline at end of file
+?>
```

Comment:
Please read the guide on contributing to suitecrm. https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM

It is best to create a branch from the develop branch for features. Or create a branch from hotfix for issues / fixes. I this case you should create a branch from hotfix.